src/sage/rings/polynomial/polynomial_ring.py

```diff
@@ -2483,17 +2482,14 @@ def fraction_field(self):
             sage: t(x)
             x
 
+        Issue :issue:`37374`::
```

Comment:
```suggestion
        :issue:`37374`::
```
This is redundant, including in the compiled doc.